requirements_pt.txt

```diff
@@ -1,8 +1,8 @@
 numba
-numpy < 2.0
-peft
-prettytable
-psutil
-py-cpuinfo
-pydantic
+numpy==1.23.5 ; python_version < '3.12'
```

Comment:
Could you clarify the reason for adding strict version limitations on these packages, particularly for `prettytable`, `psutil`, `py-cpuinfo`, and `pydantic`?

Reply:
These version limitation is from Habana team  for the robust. @chensuyue do you think we should follow that?